audio/out/ao_pipewire.c

```diff
@@ -489,7 +489,10 @@ static int pipewire_init_boilerplate(struct ao *ao)
     if (pw_thread_loop_start(p->loop) < 0)
         goto error;
 
-    context = pw_context_new(pw_thread_loop_get_loop(p->loop), NULL, 0);
+    context = pw_context_new(
+            pw_thread_loop_get_loop(p->loop),
+            pw_properties_new(PW_KEY_CONFIG_NAME, "client-rt.conf", NULL),
```

Comment:
The context takes ownership of the properties? The pipewire docs don't say.

Reply:
It says so for me:
https://docs.pipewire.org/group__pw__context.html , context.h and context.c.

Reply:
Yes. Not sure why I didn't see that. Thanks.